src/components/QuickOpenModal.js

```diff
@@ -332,21 +340,26 @@ export class QuickOpenModal extends Component<Props, State> {
     };
     const html = fuzzyAldrin.wrap(candidateString, query, options);
     return <div dangerouslySetInnerHTML={{ __html: html }} />;
-  };
+  }
+
+  updateResult(result: QuickOpenResult, newQuery: string): QuickOpenResult {
+    return {
+      ...result,
+      title: this.renderHighlight(result.title, basename(newQuery), "title")
+    };
+  }
```

Comment:
i moved this to its own function to simplify the typing but i think we can move it back...